magento-research/pwa-studio
# Update devdocs dependencies

## Description

Update dependencies in `pwa-devdocs`

## Related Issue
None

## Verification Steps

1. Navigate to the `pwa-devdocs` directory
2. Run an audit: `npm audit`
3. Verify there are no errors/vulnerabilities found.

## Checklist:
